.github/workflows/ruby.yml

```diff
@@ -74,6 +75,7 @@ jobs:
       run: bundle exec rake
       env:
         SETTINGS__ILS__BIB_MODEL: SearchworksItem
+        SETTINGS__ILS__PICKUP_DESTINATION_CLASS: Symphony::PickupDestination
```

Comment:
I don't think this is necessary as it is set here: https://github.com/sul-dlss/sul-requests/pull/1638/files#diff-e769bbb8c1ba3711c5403b424ed9c218ffafba7f1890ee394717196f28ff4540R86

Reply:
Ok, I'll remove that one and try it. I wasn't sure what was required to get the tests to pass, and the FOLIO part was required.